webapp/src/shared/components/Header/Header.tsx

```diff
@@ -5,34 +5,52 @@ import Icon, { IconSize, IconVariant } from '../Icon/Icon';
 import Status, { StatusVariant } from '../Status/Status';
 import Text, { TextColor, TextVariant, TextWeight } from '../Text/Text';
 import './Header.css';
+import React from 'react';
 
-type HeaderProps = {
+type HeaderCommon = {
   navigationFigure: IconVariant | string;
-  navigationUrl: string;
   statusVariant?: StatusVariant;
   children: string;
 };
 
+type HeaderLinkProps = HeaderCommon & {
```

Comment:
We need a button to go back here, so I changed it on the template of the change I also made for the row component.